### ISSUE TITLE
Referenced image in v0.7.0 does not exist

### ISSUE THREAD

**Issue by megian** [Xelon-AG/xelon-csi#22]
https://github.com/Xelon-AG/xelon-csi/blob/main/deploy/kubernetes/releases/xelon-csi-v0.7.0/driver.yml#L199
```
$ docker pull xelonag/xelon-csi:v0.7.1
Error response from daemon: manifest for xelonag/xelon-csi:v0.7.1 not found: manifest unknown: manifest unknown
```

**Comment by pavel-github** [comment]
@megian, nice catch. Thanks for reporting.
It is a typo and should be `v0.7.0`.

**Comment by pavel-github** [comment]
Fixed with f1de93537bc32b6bf28da12d07ae3305634593ca.
